content/docs/components/serving/seldon.md

```diff
@@ -6,72 +6,18 @@ weight = 40
 
 ## Serve a model using Seldon
 
-_This section has not yet been converted to kustomize, please refer to [kubeflow/manifests/issues/10](https://github.com/kubeflow/manifests/issues/10)._
+Seldon comes installed with Kubeflow.
 
-[Seldon-core](https://github.com/SeldonIO/seldon-core) provides deployment for any machine learning runtime that can be [packaged in a Docker container](https://docs.seldon.io/projects/seldon-core/en/latest/wrappers/README.html).
+The documentation for using Seldon can be found [here](https://docs.seldon.io/projects/seldon-core/en/latest/).
```

Comment:
For clarity and accessibility (e.g. in screen readers) it's best to give more information than just `here` in link alias text. 

Suggestion:

```
You can find full instructions in the [Seldon docs](https://docs.seldon.io/projects/seldon-core/en/latest/).
```